pkg/storage/loaders/mto_common_solution_contractor.go

```diff
@@ -21,7 +22,7 @@ type mtoCommonSolutionContractorLoaders struct {
 
 // MTOCommonSolutionContractor is the singleton instance of all LoaderWrappers related to MTOCommonSolutionContractor
 var MTOCommonSolutionContractor = &mtoCommonSolutionContractorLoaders{
-	ByID:                NewLoaderWrapper(batchMTOCommonSolutionContractorGetByID),
+	ByID:                NewLoaderWrapper(MTOCommonSolutionContractorGetByCommonSolutionIDLoader),
```

Comment:
I think this got lost in the revert, can you reimplement the naming please?

Reply:
oops yeah got lost in the revert, refixed it now

Reply:
```suggestion
	ByID:                NewLoaderWrapper(batchMTOCommonSolutionContractorGetByID),
```